app/src/main/res/layout-land/back_arrow_toolbar.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@rt4914 When we already have dedicated toolbar why we need to introduce this ? we could have renamed that is't?

Reply:
@nikitamarysolomanpvt I haven't reviewed this PR and I think which ever-way you decide should be fine. You can directly suggest renaming the earlier file to the author.

Reply:
May I know why we are introducing new toolbar when we already have one which is used in several places...

Reply:
@NullByte08 We are planning some fix to make the implementation much simpler and clean so please keep toolbar PRs on hold. Will get back to you on this by Monday.

Reply:
@nikitamarysolomanpvt You are right using only one toolbar file will be much cleaner. #1001 demanded to make a toolbar file for that specific design, hence I made the `backarrowtoolbar.xml`. But, if you say so I will replace all the files with the existing `toolbar.xml` and set up the back arrow homeAsUpIndicator in the kotlin file itself. Whichever way you decide is completely fine. :)

Reply:
@nikitamarysolomanpvt please clarify on this.

Reply:
> @nikitamarysolomanpvt You are right using only one toolbar file will be much cleaner. #1001 demanded to make a toolbar file for that specific design, hence I made the `backarrowtoolbar.xml`. But, if you say so I will replace all the files with the existing `toolbar.xml` and set up the back arrow homeAsUpIndicator in the kotlin file itself. Whichever way you decide is completely fine. :)

@NullByte08  You are right thats what am thinking and also i was proposing  that having a viewModel or class with methods to do this would avoid boilerplates. We are having dedicated class for handling status bar color as well... Just waiting for the reply of @BenHenning on Object composition.

Reply:
@NullByte08 Please create a view model and also use the existing toolbar. We may control the icon , title and so on by viewmodel

Reply:
@nikitamarysolomanpvt Ok